src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -268,7 +268,7 @@ def _construct_job_output_state_set(
 
         return output_states
 
-    def lookup_job_info(self, job_ids: List[str]) -> dict:
+    def get_job_info(self, job_ids: List[str]) -> dict:
```

Comment:
renamed to be more in line with other similar functions -- e.g. `_get_job_ids_by_cell_id` or `_get_jog_logs`